portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,28 @@
     <script src="script.js"></script>
   </head>
   <body>
+    
+    <!-- Navigation Bar -->
+    <div class = "flex-nav-bar">
+        <p class="nav-item"> Home </p>
+        <p class="nav-item"> Projects </p>
+        <p class="nav-item arrow-link"> Github&rarr; </p>
+    </div>
+
+    <!-- Main container -->
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <img src="/images/avataaars.svg">
+
+        <!-- Display box container-->
+        <div class = "description">
+            <h2> Hello there 👋 </h2>
```

Comment:
Avoiding using` <h1><h2>` for styling purpose here,  checkout "semantic html".
Use `<p> </p>` and style them later with css.